compose/foundation/foundation/src/desktopMain/kotlin/androidx/compose/foundation/TooltipArea.desktop.kt

```diff
@@ -124,15 +123,18 @@ fun TooltipArea(
         isVisible = false
     }
 
-    fun hideIfNotHovered(globalPosition: Offset) {
-        if (!parentBounds.contains(globalPosition)) {
+    fun hideIfNotHovered(localBounds: LayoutBoundsHolder, localPointerPosition: Offset) {
+        val offsetInRoot = localBounds.bounds?.positionInRoot ?: return
+        val pointerPositionInRoot = offsetInRoot + localPointerPosition.round()
```

Comment:
Why are we round it?

Reply:
> Theoretically it's possible

Also may be in case of transformation. We can scale the tooltip, and all pointers should also be scaled

> I can use floats everywhere instead

I thought that these are the same methods as in the previous code that return float. But I looked at the code and they are new, and they are int. So no point to do that.

Reply:
> Also may be in case of transformation. We can scale the tooltip, and all pointers should also be scaled

Do you mean `graphicsLayer` scaling, or...? What scaling affects pointer locations?